src/universal/components/CreateCard/CreateCard.js

```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```

Comment:
but if someone increases their default browser font size, then this means the corners will look more rounded, right?